onnx/checker.py

```diff
@@ -8,197 +8,59 @@
 from __future__ import print_function
 from __future__ import unicode_literals
 
-import itertools
-import logging
+import functools
 
-from onnx.onnx_pb2 import TensorProto, ValueInfoProto, NodeProto, AttributeProto,  \
-    GraphProto, ModelProto, IR_VERSION
-from onnx import defs, mapping
+from onnx import (ValueInfoProto,
+                  AttributeProto,
+                  TensorProto,
+                  NodeProto,
+                  GraphProto,
+                  ModelProto,
+                  IR_VERSION)
+import onnx.onnx_cpp2py_export.checker as C
 
 
+def _create_checker(proto_type):
+    def decorator(py_func):
+        @functools.wraps(py_func)
+        def checker(proto, ir_version=IR_VERSION):
+            if not isinstance(proto, proto_type):
+                raise RuntimeError(
```

Comment:
ValueError?

Reply:
yep indeed ValueError should be raised for bad arguments